unit-tests/py/rspy/devices.py

```diff
@@ -77,19 +70,9 @@ def __init__( self, sn, dev ):
         if dev.supports( rs.camera_info.product_line ):
             self._product_line = dev.get_info( rs.camera_info.product_line )
         self._physical_port = dev.supports( rs.camera_info.physical_port ) and dev.get_info( rs.camera_info.physical_port ) or None
-        self._usb_location = None
-        try:
-            self._usb_location = _get_usb_location( self._physical_port )
-        except Exception as e:
-            log.e( 'Failed to get usb location:', e )
-        self._port = None
-        if acroname:
-            try:
-                self._port = _get_port_by_loc( self._usb_location )
-            except Exception as e:
-                log.e( 'Failed to get device port:', e )
-                log.d( '    physical port is', self._physical_port )
-                log.d( '    USB location is', self._usb_location )
+
+        self._usb_location, self._port = usb_relay.get_usb_and_port_location(relay, self._physical_port, _hubs)
```

Comment:
Why not `relay.get_usb_and_port_location( ... )`?

Reply:
Because it's a static function of the module usb_relay, not an instance function

Reply:
But you pass the relay to it. Why is it not a hub method? The caller doesn't and shouldn't know...